### PR TITLE
Don't show empty pages on project nav

### PR DESCRIPTION
**Pull request by edpaget**
If someone created a page but hasn't added content don't show it

Closes zooniverse/Panoptes#1100

### CODE REVIEW

**Comment by chrissnyder** [comment]
The `project-research` Link still shows, even if it's empty. Is this intended?

File: app/pages/project/index.cjsx [hunk 1]
```diff
@@ -75,7 +75,8 @@ ProjectPage = React.createClass
               <Translate content="project.nav.research" />
             </Link>
             <PromiseRenderer promise={@props.project.get 'pages'}>{(pages) =>
-              pageTitles = pages.reduce(((accum, page) -> accum[page.url_key] = page.title; accum), {})
+              pageTitles = pages.filter((page) -> page.content isnt '' and page.content?).reduce(((accum, page) -> accum[page.url_key] = page.title; accum), {})
+              console.log(pageTitles)
```
Comment (chrissnyder) [inline app/pages/project/index.cjsx]: Nit pick, but could you remove the log?

Comment (edpaget) [inline app/pages/project/index.cjsx]: Sorry removed

**Comment by edpaget** [comment]
Yes it is. That page will have the team list even if there isn't any content on it.

**Comment by chrissnyder** [comment]
Gotcha. Works for me.

**Pull request merged**